scripts/flyway/configdb/V1.0.0__initialization.sql

```diff
@@ -46,7 +46,7 @@ DROP TABLE IF EXISTS `AppNamespace`;
 CREATE TABLE `AppNamespace` (
   `Id` int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT '自增主键',
   `Name` varchar(32) NOT NULL DEFAULT '' COMMENT 'namespace名字，注意，需要全局唯一',
-  `AppId` varchar(32) NOT NULL DEFAULT '' COMMENT 'app id',
+  `AppId` varchar(64) NOT NULL DEFAULT '' COMMENT 'app id',
```

Comment:
To make flyway work, we need to create a new file with an increment version, such as `V1.2.0__appid_extension.sql`.

```sql
Use ApolloConfigDB;
alter table `AppNamespace`  change AppId AppId varchar(64) NOT NULL DEFAULT 'default' COMMENT 'app id';
alter table `Cluster`  change AppId AppId varchar(64) NOT NULL DEFAULT 'default' COMMENT 'app id';
alter table `GrayReleaseRule`  change AppId AppId varchar(64) NOT NULL DEFAULT 'default' COMMENT 'app id';
alter table `Instance`  change AppId AppId varchar(64) NOT NULL DEFAULT 'default' COMMENT 'app id';
alter table `InstanceConfig` change ConfigAppId ConfigAppId varchar(64) NOT NULL DEFAULT 'default' COMMENT 'Config App Id';
alter table `ReleaseHistory`  change AppId AppId varchar(64) NOT NULL DEFAULT 'default' COMMENT 'app id';
```

Reply:
Yes, I forgot it. I'll commit later.